core/meson.build

```diff
@@ -47,6 +48,13 @@ cogcore_dependencies = [
 
 install_headers(cogcore_headers, cogcore_config_h, subdir: 'cog')
 
+if (wpe_dep.version().version_compare('>= 1.13.3') and manette_dep.found())
```

Comment:
Probably we want to check for version `>=1.13.90` here, to be sure the latest modifications to the API are available.

Reply:
done